app/views/testing_grounds/_technologies_form.html.haml

```diff
@@ -32,7 +32,11 @@
       %span.check_box
         - max = maximum_concurrency?(technology.key, testing_ground.technology_profile)
         = check_box_tag technology.key, "max", max, {data: {toggle: "toggle", on: "Max", off: "Min"}}
-        %span= technology.name
+        %span
+          - if technology.composite?
+            = technology.name
+          - else
+            = technology.name.pluralize
```

Comment:
This would be ideally done using the locale YAML files, such as:

```yaml
technologies:
  transport_car_using_electricity:
    singular: Electric car
    plural: Electric cars
  buffer_space_heating:
    singular: Buffer space heating
    plural: Buffer space heating
```

```haml
%span= t("technologies.#{ technology.key }.plural")
```